converter/ena/classes/sra_experiment.py

```diff
@@ -0,0 +1,784 @@
+from dataclasses import dataclass, field
```

Comment:
Are we copying these classes from somewhere? They seem like very big files..

Reply:
these are generated from ENA XSDs for experiment and run XMLs.

Reply:
I see. It would be good to add instructions on how to regenerate them in the readme

Reply:
added instruction in the respective module.